Aztec/Classes/Formatters/FontFormatter.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import UIKit
+
+class FontFormatter: CharacterAttributeFormatter {
+
+    let elementType: Libxml2.StandardElementType
+    
+    let traits: UIFontDescriptorSymbolicTraits
+
+    init(elementType: Libxml2.StandardElementType, traits: UIFontDescriptorSymbolicTraits) {
+        self.elementType = elementType
+        self.traits = traits
+
+    }
+
+    func apply(to attributes: [String : Any]) -> [String: Any] {
```

Comment:
This is a really good opportunity to add more unit tests, since these methods are so atomic.

Consider adding: 1 unit test adding an attribute that's not in the provided list, and 1 unit test to add an attribute that's already in there.